### ISSUE TITLE
unclear how to use additional classpath

### ISSUE THREAD

**Issue by GianniGiglio** [GroovyLanguageServer/groovy-language-server#81]
Hi would it be possible to explain how I can add additional classpath entries form jars. It's unclear how this exactly works.

**Comment by GianniGiglio** [comment]
@joshtynjala That's what I don't understand how to set this setting. Could you explain that part?

**Comment by joshtynjala** [comment]
It's an array of file system paths. You set it with the same values that you might pass to the `-cp`/`-classpath` compiler option.

**Comment by GianniGiglio** [comment]
I am feeling pretty stupid I still don't understand your answer.

tried this but it didn't work

java -cp libs/ -jar groovy-lang-orignal-all.jar

**Comment by himanshutripathi01** [comment]
if you are using visual studio code 
Open workspace settings 
and then append this to the json

{
    "groovy.classpath": [
        "io/2.11.0/a2503f302b11ebde7ebc3df41daebe0e4eea3689/commons-io-2.11.0.jar",
	    "/Users//.gradle/caches/modules-2/files-2.1/org.apache.commons/commons-csv/1.9.0/b59d8f64cd0b83ee1c04ff1748de2504457018c1/commons-csv-1.9.0.jar",
    ],

}

**Comment by GianniGiglio** [comment]
@himanshutripathi01 not using vscode but codemirror languag server

**Comment by GianniGiglio** [comment]
I think it starts making sence now. Could it be I have to add this settings file to the workspace of the language server?

**Comment by himanshutripathi01** [comment]
yes
